src/Tests/Tests.csproj

```diff
@@ -22,6 +22,10 @@
         </PackageReference>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == 'NET8_0_OR_GREATER'">
```

Comment:
Does this check really work? I thought NET8_0_OR_GREATER is a preprocessor symbol that can only be used in C#, but not in the csproj

Reply:
Unclear I had trouble because of my analyzer settings. I really should mark this as draft until I can resolve.